LayoutTests/css3/masking/mask-reference-png-fragment.html

```diff
@@ -0,0 +1,31 @@
+ <style>
+    .container {
+        display: inline-block;
+        border: 1px solid black;
+    }
+    .box {
+        background-color: green;
+        width: 100px;
+        height: 100px;
+    }
+    .green-background {
+        background-color: green;
+    }
+    .blue-background {
+        background-color: blue;
+    }
+    .mask-bitmap-top-fragment {
+        mask-image: url(resources/red-100x100.png#xywh=0,0,100,50);
```

Comment:
We don't support this fragment syntax, right?

Reply:
Yes we do not support it right now. So I added this test. marked it as `[ ImageOnlyFailure ]` and filed [261667](https://bugs.webkit.org/show_bug.cgi?id=261667) to track this issue till it is fixed.